tasks/task375_classify_type_of_sentence_in_debate.json

```diff
@@ -0,0 +1,2541 @@
+{
+    "Contributors": [
+        "Pegah Alipoormolabashi"
+    ],
+    "Source": [
+        "https://aclanthology.org/2020.emnlp-main.716.pdf",
+        "The instances for which there was a majority vote on a class label."
+    ],
+    "Categories": [
+        "Classification"
+    ],
+    "Definition": "In this task, you will be given a debate topic, along with a sentence from the debate. You should classify the given sentence and choose the type of that sentence. Possible types are explained below.\nPolicy: This refers to a sentence proposing a specific course of action to be taken. It typically contains modal verbs like \"should\" and \"ought to\". It cannot be directly proved with objective evidence, and a proper type of support is a logical reason from which the proposition can be inferred.\nValue: This refers to a sentence containing value judgments without making specific claims about what should be done (If so, then it is a Policy sentence.). Because of the subjectivity of value judgments, it cannot be proved directly with objective evidence.\nFact: This refers to an objective proposition expressing or dealing with facts or conditions as perceived without distortion by personal feelings, prejudices, or interpretations. A Fact sentence has a truth value that can be verified with objective evidence that may be available at the time the claim is made; predictions about future are considered unverifiable.\nTestimony: This refers to an objective sentence about the author's personal state or experience. Evidence for Testimony is not publicly available in most cases.\nReference: This refers to a reference to a source of objective evidence. It is typically a citation or a URL of a published work from a renowned source.",
+    "Positive Examples": [
+        {
+            "input": "topic: 4a. Campaign Finance Reform sentence: As it turns out, the laws we have work and people don't forgive those in the pockets of corporations.",
+            "output": "Fact",
+            "explanation": "This sentence is an objective proposition stating a fact."
+        },
+        {
+            "input": "topic: Abortion Is Bad sentence: It also bad because God made all of us so only he can end our lives.",
+            "output": "Value",
+            "explanation": "The sentence is expressing a value judgement without suggesting a policy."
+        }
+    ],
```

Comment:
Wondering if you can add more examples here? (like at least one example from each category).

Reply:
sure! ok.